.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  pull_request:
+    tags-ignore: ["*"]
```

actionlint (1.7.7)

"tags-ignore" filter is not available for pull_request event. it is only for push event

(events)